test/gui/shared/steps/steps.py

```diff
@@ -350,3 +350,16 @@ def step(context):
         waitForObjectExists(names.oCC_ShareLinkWidget_checkBox_password_QProgressIndicator).visible,
         False
     )
+  
+@When('user "|any|" opens the sharing dialog of "|any|" to delete the folder shared using the client-UI')
```

Comment:
it really does not matter in this step for what reason the user opens the sharing dialog. the reason and what will happen in that dialog will be determent in the next steps

```suggestion
@When('user "|any|" opens the sharing dialog of "|any|" using the client-UI')
```